superset/migrations/versions/2024-04-08_15-43_5ad7321c2169_mig_new_csv_upload_perm.py

```diff
@@ -0,0 +1,85 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""mig new csv upload perm
+
+Revision ID: 5ad7321c2169
+Revises: 678eefb4ab44
+Create Date: 2024-04-08 15:43:29.682687
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "5ad7321c2169"
+down_revision = "678eefb4ab44"
+
+from alembic import op
+from sqlalchemy.exc import SQLAlchemyError
+from sqlalchemy.orm import Session
+
+from superset.migrations.shared.security_converge import (
+    add_pvms,
+    get_reversed_new_pvms,
+    get_reversed_pvm_map,
+    migrate_roles,
+    Pvm,
+)
+
+NEW_PVMS = {"Database": ("can_csv_upload",)}
+
+PVM_MAP = {
+    Pvm("CsvToDatabaseView", "can_this_form_post"): (
+        Pvm("Database", "can_csv_upload"),
+    ),
+    Pvm("CsvToDatabaseView", "can_this_form_get"): (Pvm("Database", "can_csv_upload"),),
+}
+
+
+def do_upgrade(session: Session) -> None:
+    add_pvms(session, NEW_PVMS)
```

Comment:
I guess you could say it's a breaking change, on `UPDATING.md` should I add a note to `## Next` or `### Breaking changes` ?

Reply:
The problem is that if this PR contains breaking changes, it will be blocked until the next breaking window. There are two ways to avoid this:
- Make it backward compatible
- Remove the permission changes from this PR and do them during the breaking window